LibreNMS/Snmptrap/Handlers/LinkDown.php

```diff
@@ -52,9 +52,17 @@ public function handle(Device $device, Trap $trap)
             return;
         }
 
-        $port->ifOperStatus = $trap->getOidData("IF-MIB::ifOperStatus.$ifIndex");
-        $port->ifAdminStatus = $trap->getOidData("IF-MIB::ifAdminStatus.$ifIndex");
+        $trapOperStatus = $trap->getOidData("IF-MIB::ifOperStatus.$ifIndex");
+        if ($trapOperStatus) {
+            $port->ifOperStatus = $trapOperStatus;
+        } else {
+            $port->ifOperStatus = 'down';
+        }
```

Comment:
Hi @tuomari 
You could make it shorter with something like this: 
`$port->ifOperStatus = $trapOperStatus ?? 'down';`